### PR TITLE
fix batching of changes in doctypemigration continuous

### PR DESCRIPTION
**Pull request by dannyroberts**
was committing every change before

### CODE REVIEW

File: corehq/doctypemigrations/continuous_migrate.py [hunk 1]
```diff
@@ -78,5 +78,5 @@ def _get_time_since_last_commit(self):
         return datetime.datetime.utcnow() - self._last_commit_time
 
     def should_commit(self):
-        return (self._uncommitted_changes_count < self.max_changes_before_commit or
```
Comment (benrudolph) [inline corehq/doctypemigrations/continuous_migrate.py]: haha

**Comment by benrudolph** [comment]
:+1: after tests

**Comment by benrudolph** [comment]
(also feel free to self merge if i forget)

**Comment by czue** [comment]
:+1: are there any tests for this? if not, go ahead and merge

**Comment by dannyroberts** [comment]
There are tests for code around this though not specifically for batching. I think it's conceivable that tests could catch something (and I'm in no rush) so I'll wait it out

**Comment by dannyroberts** [comment]
canceling the tests for remove print

**Pull request merged**